distribution/lib/Standard/Table/0.0.0-dev/src/Column.enso

```diff
@@ -243,15 +245,16 @@ type Column
     == : Column | Any -> Column
     == self other =
         new_name = naming_helper.binary_operation_name "==" self other
-        fallback problem_builder a b =
-            if (a.is_a Float) || (b.is_a Float) then
-                problem_builder.reportFloatingPointEquality -1
-            a == b
-        run_vectorized_binary_op_with_fallback_problem_handling self Java_Storage.Maps.EQ other fallback_fn=fallback new_name=new_name expected_result_type=Value_Type.Boolean skip_nulls=True
+        call_comparator self other new_name Comparators.Eq <|
+            fallback problem_builder a b =
+                if (a.is_a Float) || (b.is_a Float) then
+                    problem_builder.reportFloatingPointEquality -1
+                a == b
```

Comment:
Something seems off about this block, if I understand correctly it translates to
```
        call_comparator self other new_name Comparators.Eq <|
            fallback = problem_builder-> a-> b-> =
                if (a.is_a Float) || (b.is_a Float) then
                    problem_builder.reportFloatingPointEquality -1
                a == b
```
so the block after `<|` actually discards the `fallback` (tbh we should be getting an 'unused variable' warning) and returns `Nothing`.

That does not seem what was meant.

Reply:
No fallback is passed to the `run_vectorized_binary_op_with_fallback_problem_handling`